doc/how-to/recover.md

```diff
@@ -0,0 +1,68 @@
+# How to recover a MicroCloud cluster
+
+```{note}
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
+database for highly-available storage. While the cluster recovery process is
+similar for each service, this document only covers cluster recovery for the
+`microcloudd` daemon. For cluster recovery procedures for LXD, MicroCeph and
+MicroOVN, see:
+
+- [LXD Cluster Recovery](https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_recover/)
+- [MicroOVN Launchpad Bug](https://bugs.launchpad.net/microovn/+bug/2072377)
+- [MicroCeph Issue](https://github.com/canonical/microceph/issues/380)
+```
+
+Dqlite requires a majority of its database voters (a quorum) to be accessible in
```

Comment:
Hm, "a majority of its database voters" can sound very misleading. I believe we cap out at 4 voters at a time, so this might make someone think they need at least 3 nodes up at all times, when really it's 2.

How about this:
```
For MicroClouds with 2 or more systems, at least 2 database voters (a quorum) must be accessible in order to perform database operations
```

Reply:
Hmm, I thought I recalled at least LXD hitting 4 voters, 2 standby, 1 spare but you're right, both LXD and microcluster have the max voters set to 3. 

In that case I take it back, "majority" is correct, because we may configure it someday and have to change the docs. Sorry for the mislead.